README.md

```diff
@@ -173,8 +173,6 @@ We also use the issue tracker to track features. If you have an idea for a featu
 
 [zoom_meeting]: https://zoom.us/j/92253194848?pwd=cVVVNDMxeTl1QVJPUlpvLzNSVU1JZz09
 
-[time_zone_converter]: http://www.thetimezoneconverter.com/?t=08:00&tz=PT%20%28Pacific%20Time%29
```

Comment:
This only allowed me to pin to either summer or winter time. So I dropped it to avoid confusion